cluster-operator/src/main/resources/cluster-roles/020-ClusterRole-strimzi-cluster-operator-role.yaml

```diff
@@ -8,7 +8,6 @@ rules:
 - apiGroups:
   - ""
   resources:
-  # The cluster operator needs to access and manage service accounts to grant Strimzi components cluster permissions
```

Comment:
I think all these changes in the RBAC files are because you are rewriting some other changes done while the PR was opened. Maybe you need torefesh the Helm3 files?